exec/java-exec/src/test/java/org/apache/drill/exec/sql/TestAnalyze.java

```diff
@@ -332,7 +332,7 @@ public void testUseStatistics() throws Exception {
       query = " select emp.employee_id from dfs.tmp.employeeUseStat emp join dfs.tmp.departmentUseStat dept"
               + " on emp.department_id = dept.department_id "
               + " group by emp.employee_id";
-      String[] expectedPlan8 = {"HashAgg\\(group=\\[\\{0\\}\\]\\).*rowcount = 730.2832515526484,.*",
+      String[] expectedPlan8 = {"HashAgg\\(group=\\[\\{0\\}\\]\\).*rowcount = 730.2832515526.*",
```

Comment:
Did we drop the last three digits here to prevent spurious test failures resulting from rounding errors?

Reply:
Calcite has slightly updated the formula to calculate it. I have removed these digits to avoid failures for further potential updates.